api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2MirrorSpec.java

```diff
@@ -48,6 +52,8 @@ public void setTopicsPattern(String topicsPattern) {
         this.topicsPattern = topicsPattern;
     }
 
+    @DeprecatedProperty(movedToPath = "topicsExcludePattern")
```

Comment:
I think in the other cases we use this we use full path to the field? E.g. `.spec.mirrors.topicsExcludePattern`? It would be good to use the same.

Reply:
Sure